dev-python/smart-open/smart-open-6.3.0.ebuild

```diff
@@ -24,12 +24,13 @@ RDEPEND="
 	dev-python/paramiko[${PYTHON_USEDEP}]
 "
 
-BDEPEND="
-	test? (
-		dev-python/moto[${PYTHON_USEDEP}]
-		dev-python/responses[${PYTHON_USEDEP}]
-		dev-python/pytest-rerunfailures[${PYTHON_USEDEP}]
-	)
-"
+#BDEPEND="
+#	test? (
+#		<dev-python/moto-5[${PYTHON_USEDEP}]
+#		dev-python/responses[${PYTHON_USEDEP}]
+#		dev-python/pytest-rerunfailures[${PYTHON_USEDEP}]
+#	)
+#"
```

Comment:
We don't have to comment out the deps, restricting the test phase is sufficient.